### PR TITLE
Fix workspace status indication

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix workspace status indication for the "Start workspace" page.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/19055

Screenshot from 2021-03-01 15-20-19
Screenshot from 2021-03-01 15-21-04

### CODE REVIEW

**Comment by sleshchenko** [comment]
For me the screenshots from PR description are not clear, the references issue does not have any issue with Error state.
But I see that PR makes situation better, stopping workspaces are displayed as `stopping` but it would be ideally if we align it with other statuses: Stopped, Starting
Screenshot_20210301_173201
Screenshot_20210301_173217

File: src/components/WorkspaceStatusLabel/index.module.css [hunk 1]
```diff
@@ -2,6 +2,12 @@
   padding-right: 4px;
 }
 
+.defaultLabel,
+.stoppedLabel {
```
[the remark this answers is not in the document]
Comment (olexii4) [reply src/components/WorkspaceStatusLabel/index.module.css]: I have fixed

File: src/pages/IdeLoader/index.tsx [hunk 2]
```diff
@@ -258,7 +259,7 @@ class IdeLoader extends React.PureComponent<Props, State> {
           </AlertGroup>
         )}
         <Header title={`Starting workspace ${workspaceName}`}
-          status={WorkspaceStatus[hasError ? WorkspaceStatus.ERROR : WorkspaceStatus.STARTING]} />
+          status={status} />
```
[the remark this answers is not in the document]
Comment (olexii4) [reply src/pages/IdeLoader/index.tsx:262]: I have fixed it.

**Comment by JPinkney** [comment]
[dashboard-next-ci-test]

**Review by JPinkney — APPROVED** [review]
Just re-tested with the changes and it appears that it's working for me now

**Review by sleshchenko — APPROVED** [review]
I did not retested but it worked fine for me last time and stopping -> Stopping should be fixed now

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**